master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -1105,7 +1105,8 @@ private[celeborn] class Master(
           (statusSystem.excludedWorkers.asScala ++ statusSystem.manuallyExcludedWorkers.asScala).asJava),
         needCheckedWorkerList,
         new util.ArrayList[WorkerInfo](
-          (statusSystem.shutdownWorkers.asScala ++ statusSystem.decommissionWorkers.asScala).asJava)))
+          (statusSystem.shutdownWorkers.asScala ++ statusSystem.decommissionWorkers.asScala).asJava),
+        CheckQuotaResponse(isAvailable = true, "")))
```

Comment:
should actually check quota for the application.

Reply:
Currently QuotaManager only support check quota for user, at next PR, we will support check quota for application, so we don’t need to modify rpc proto at this PR.